snomed/com.b2international.snowowl.snomed.reasoner/src/com/b2international/snowowl/snomed/reasoner/request/ClassificationCreateRequest.java

```diff
@@ -101,8 +103,18 @@ public String execute(final BranchContext context) {
 				.addAllConcepts(additionalConcepts)
 				.build(branch.path());
 		
+		CodeSystem codeSystem = CodeSystemRequests.prepareSearchCodeSystem()
+				.build()
+				.execute(context)
+				.getItems()
+				.get(0);
+		
+		int maxReasonerCount = codeSystem.getSettings().containsKey(SnomedCoreConfiguration.MAXIMUM_REASONER_COUNT)
+				? (int) codeSystem.getSettings().get(SnomedCoreConfiguration.MAXIMUM_REASONER_COUNT)
+				: SnomedCoreConfiguration.DEFAULT_MAXIMUM_REASONER_COUNT;
```

Comment:
Most of these lookups would benefit from `Map.getOrDefault(key, defaultValue)`:

```suggestion
		int maxReasonerCount = codeSystem.getSettings().getOrDefault(
			SnomedCoreConfiguration.MAXIMUM_REASONER_COUNT,
			SnomedCoreConfiguration.DEFAULT_MAXIMUM_REASONER_COUNT);
```